heka/files/lua/common/afd.lua

```diff
@@ -128,9 +128,18 @@ function reset_alarms()
     alarms = {}
 end
 
+local disallowed_dimensions = {
+    name = true,
+    value = true,
+    hostname = true,
+    member = true,
+    no_alerting = true,
+    tag_fields = true,
+}
```

Comment:
shouldn't this be treated and caught in _modules/alarming.py instead? Because it might be quite tricky for the deployer to understand that the dimension name is reserved.